lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -201,6 +197,10 @@ sub get_dasd_disk_management {
     return Installation::DiskActivation::DASDDiskManagementController->new();
 }
 
+sub get_license_popup {
+    return Installation::Popups::LicensePopup->new({app => YuiRestClient::get_app()});
```

Comment:
it is not needed to be passed, the base class take care of having an `app` instance for th object.